string servo
string setType
int value
